src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/Validated.container.tsx

```diff
@@ -76,8 +74,8 @@ export const Validated = ({
         dataEntryItemId: string,
         dataEntryIdArgument: string,
         formFoundationArgument: RenderFoundation,
-        saveType: ?$Values<typeof addEventSaveTypes>,
-        enrollment: ?Object,
+        saveType?: keyof typeof addEventSaveTypes,
```

Comment:
Change for `saveType?: typeof addEventSaveTypes[keyof typeof addEventSaveTypes],`